cadc-vosi/src/main/java/ca/nrc/cadc/vosi/CapGetAction.java

```diff
@@ -127,18 +125,21 @@ protected InlineContentHandler getInlineContentHandler() {
 
     @Override
     public void doAction() throws Exception {
+        if (CapInitAction.getAuthRequired(componentID)) {
+            AuthMethod am = AuthenticationUtil.getAuthMethod(AuthenticationUtil.getCurrentSubject());
+            if (am != null && am.equals(AuthMethod.ANON)) {
```

Comment:
If am==null here wouldn't that also mean anonymous?

Reply:
yes, hard to test because I don't think that happens in our code right now, but will fix anyway.